src/gtk/webview_webkit.cpp

```diff
@@ -949,10 +949,12 @@ wxString wxWebViewWebKit::GetPageText() const
                     wxConvUTF8);
 }
 
-void wxWebViewWebKit::RunScript(const wxString& javascript)
+bool wxWebViewWebKit::RunScript(const wxString& javascript, wxString* output)
 {
     webkit_web_view_execute_script(m_web_view,
                                    javascript.mb_str(wxConvUTF8));
+
+    return true;
```

Comment:
I'm not sure if it's a good idea to return `true` if the user code expects to get output and doesn't.

I think this method should return false if `output != NULL`. And this should be documented (in either case), of course.